src/main/kotlin/calculator/Application.kt

```diff
@@ -0,0 +1,10 @@
+package calculator
+
+class Application
+
+fun main(args: Array<String>) {
```

Comment:
```suggestion
fun main() {
```

Java에서의 메인 함수 (`public static void main(String[] args) { ... }`)와는 달리 kotlin의 메인 함수는 최상위 함수로 별도의 클래스 내부에 위치하지 않아요!  메서드 파라미터인 args 또한, 별도로 program arguments 속성 값을 받는 경우가 아니라면 생략할 수 있어요! 😃

Reply:
이번 미션에서는 사용하지 않았으니까 삭제하는 방향으로 진행할게요~